precompiles/dapps-staking/DappsStaking.sol

```diff
@@ -54,12 +54,16 @@ interface DappsStaking {
     /// @notice Withdraw all funds that have completed the unbonding process.
     function withdraw_unbonded() external;
 
-    /// @notice Claim one era of unclaimed staker rewards for the specifeid contract.
+    /// @notice Claim earned staker rewards for the oldest unclaimed era.
+    ///         In order to claim multiple eras, this call has to be called multiple times.
     ///         Staker account is derived from the caller address.
-    function claim_staker(address) external;
+    /// @param smart_contract smart contract address used for staking
+    function claim_staker(address smart_contract) external;
 
     /// @notice Claim one era of unclaimed dapp rewards for the specified contract and era.
-    function claim_dapp(address, uint128) external;
+    /// @param smart_contract smart contract address
+    /// @param era, The era to be claimed
```

Comment:
`era, The era`?

Reply:
it looks stupid here, but with syntax highlighter is ok. Let me rephrase it
<img width="779" alt="Screenshot 2022-07-15 at 13 56 25" src="https://user-images.githubusercontent.com/34627453/179218536-6636fd73-dd45-4299-b5e4-5a843f948ae5.png">

Reply:
@Dinonard check now